hieradata/common/common.yaml

```diff
@@ -484,3 +486,16 @@ networks:
         ensure: present
         iniface: eth1
         outiface: eth0
+
+# logrotate 
+profile::logging::logrotate::manage_logrotate:      true
+logrotate::rules:
```

Comment:
Hvordan vil dette oppføre seg på f.eks. debian som bruker andre filnavn enn f.eks. CentOS. Dette ligger i common (kan også flyttes til common/modules/profile.yaml for at det skal bli mer rett) så det vil bli satt opp automatisk på alle noder uavhengig av distro.